examples/e18_webhook/src/main.rs

```diff
@@ -0,0 +1,13 @@
+use serenity::{http::Http, model::webhook::Webhook};
```

Comment:
Making cargo fmt happy!!
```suggestion
use serenity::http::Http;
use serenity::model::webhook::Webhook;
```